app/helpers/provision_customize_helper.rb

```diff
@@ -4,12 +4,12 @@ def disable_check?(workflow)
       @edit[:new][:sysprep_enabled][0] && @edit[:new][:sysprep_enabled][0] != "disabled") ||
       (!@edit && @options && @options[:sysprep_enabled] &&
         @options[:sysprep_enabled][0] && @options[:sysprep_enabled][0] != "disabled")) &&
-      !workflow.kind_of?(ManageIQ::Providers::Redhat::InfraManager::ProvisionWorkflow)
+      !workflow.kind_of?(ManageIQ::Providers::Ovirt::InfraManager::ProvisionWorkflow)
```

Comment:
NOTE `ManageIQ::Providers::InfraManager::ProvisionWorkflow` is not an AR model with ActsAsStiLeafClass so this `kind_of?` will correctly pick up all subclasses